### ISSUE TITLE
[package] thrift/0.13.0: flex and bison should be build requirements

### ISSUE THREAD

**Issue by thkowa** [conan-io/conan-center-index#1830]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **thrift/0.13.0**
  * Operating System+version: **Linux Ubuntu 18.04**
  * Compiler+version: **GCC 7**
  * Conan version: **conan 1.25.2**
  * Python version: **Python 3.6.9**

### Conan profile 
```
Configuration for profile linux-x64:
[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=7
compiler.libcxx=libstdc++11
build_type=Debug
[options]
*:fPIC=True
*:shared=True
[build_requires]
[env]
```

### Problem
I think that _flex_ and _bison_ should be just build requirements and not normal requirements, as they are only needed when building the thrift compiler.

**Comment by thkowa** [comment]
That will also make the thrift package be able to cross-compile, as _flex_ does not need to be cross-compiled as build requirement. _flex_ cannot be cross-compiled at the moment!

**Comment by srnwk** [comment]
PR #2519 fixed this

**Comment by uilianries** [comment]
Fixed by #2519
